SPECS/sm.spec

```
@@ -1,15 +1,17 @@
-%global package_speccommit 24e197c0114396c911ff96a3041d4c6b470998a0
+%global package_speccommit 9faa87ad6b5c28edf9354b9d89b4f3c9c7638e5e
 %global usver 2.30.8
-%global xsver 2
+%global xsver 7
 %global xsrel %{xsver}%{?xscount}%{?xshash}
+# Series applies on top of v2.29.0 and includes all later tags to 2.30.7
+# after that tag the commits are in the patchqueue here.
 %global package_srccommit v2.29.0
 # -*- rpm-spec -*-
 
 
 Summary: sm - XCP storage managers
 Name:    sm
 Version: 2.30.8
-Release: %{?xsrel}.3%{?dist}
+Release: %{?xsrel}.4%{?dist}
```

```suggestion
Release: %{?xsrel}.1%{?dist}
```

Back to number 1, since the upstream release was incremented up to 7.